app/components/Button/Button.tsx

```diff
@@ -27,6 +27,8 @@ export const Button = forwardRef<HTMLButtonElement, Props>(
       ["rounded-2xl py-3 px-11 w-full mobileM:w-fit flex justify-center"]:
         size === "m",
       [styles.sizeNone]: size === "none",
+      ["bg-accent hover:bg-red-600 ease-in duration-300 !font-normal lg:py-5 lg:px-6 md:py-[13px] py-[15px] px-4 lg:text-2xl md:text-base text-sm"]:
```

Comment:
Size shouldn't contain all of the css that you post here, it should be only about size, let's refactor the prop to name it something like `variant`, because `size` is about dimensions of the thing, rather than everything)